### PR TITLE
add: vars

### PR DESCRIPTION
**Pull request by Myrhoiazov**

### CODE REVIEW

File: src/sass/utils/_vars.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Основной цвет шрифта, кнопок, логотипа
+$brand-color: #D41443;
+$brand-accente: #FFA5BA;
```
Comment (serovsergey) [inline src/sass/utils/_vars.scss:3]: $brand-accent

File: src/sass/utils/_vars.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+// Основной цвет бекграунда
+$bc-color: #FFB8CA;
+$bc-light: #FFF5F6;
+$bc-cafe: #F0D1A5;
+$bc-green: #C2E297;
+
+
+// Основной текст описания
+$text: #000000;
```
Comment (serovsergey) [inline src/sass/utils/_vars.scss:13]: $text-color

File: src/sass/utils/_vars.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+$subtext: #907E82;
```
Comment (serovsergey) [inline src/sass/utils/_vars.scss:14]: $subtext-color

File: src/sass/utils/_vars.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// Основной цвет шрифта, кнопок, логотипа
+$brand-color: #D41443;
+$brand-accente: #FFA5BA;
+
+// Основной цвет бекграунда
+$bc-color: #FFB8CA;
```
Comment (serovsergey) [inline src/sass/utils/_vars.scss:6]: $bg-color

**Review by serovsergey — COMMENTED** [review]
change $bc-... to $bg-...

File: src/sass/utils/_vars.scss (hunk 1 above)
Comment (Myrhoiazov) [reply src/sass/utils/_vars.scss]: +

**Review by serovsergey — APPROVED** [review]

File: src/sass/utils/_vars.scss (hunk 2 above)
Comment (Myrhoiazov) [reply src/sass/utils/_vars.scss]: +

**Pull request merged**